### PR TITLE
[BOLT] Tidy up code

### PR DESCRIPTION
**Pull request by Pribess**
Addressing #81441 #91195

### CODE REVIEW

File: bolt/runtime/hugify.cpp [hunk 1]
```diff
@@ -36,7 +36,6 @@ static void getKernelVersion(uint32_t *Val) {
   // release should be in the format: %d.%d.%d
   // major, minor, release
   struct UtsNameTy UtsName;
-  int Ret = __uname(&UtsName);
```
Comment (aaupov) [inline bolt/runtime/hugify.cpp:39]: Can't remove setting UtsName

**Review by aaupov — CHANGES_REQUESTED** [review]

**Comment by aaupov** [comment]
Can you please split up the two changes? As per LLVM policy:
> Each change should be kept as small as possible. This simplifies your work (into a logical progression), simplifies code review and reduces the chance that you will get negative feedback on the change. Small increments also facilitate the maintenance of a high quality code base.

File: bolt/runtime/hugify.cpp (hunk 1 above)
Comment (Pribess) [reply bolt/runtime/hugify.cpp]: I just updated it. Could you re-review it please?

**Comment by Pribess** [comment]
> Can you please split up the two changes? As per LLVM policy:
> 
> > Each change should be kept as small as possible. This simplifies your work (into a logical progression), simplifies code review and reduces the chance that you will get negative feedback on the change. Small increments also facilitate the maintenance of a high quality code base.

You mean i should split up commits to two PR?

**Pull request closed without merging**